setup.py

```diff
@@ -26,6 +26,7 @@
         "pyyaml>=5.4.1",
         "mlflow>=1.20.2",
         "dulwich>=0.20.26",
+        "requests>=2.21.0",
```

Comment:
Crazy alternative: what if we unfreeze all our project dependencies?   I don't have much experience here, but after some googling I found that the [packaging docs](https://packaging.python.org/discussions/install-requires-vs-requirements/#install-requires) say:

> It is not considered best practice to use install_requires to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

I've been using `>=` because pinning was causing conflicts with other packages when running, e.g., the Acme agents.  We could also consider pinning to major versions (e.g. `requests>=2,<3`).  Thoughts?

Reply:
Going to punt on this one for now.  Added issue https://github.com/agentos-project/agentos/issues/202.